.github/workflows/release-please.yml

```diff
@@ -7,7 +7,7 @@ env:
   REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository }}
   # Default minimum version of Go to support.
-  DEFAULT_GO_VERSION: "~1.20"
+  DEFAULT_GO_VERSION: "~1.21"
```

Comment:
It might be better to use the version defined in the `go.mod` file instead of manually defining them per file.

Here's an example from the codegen repo.

https://github.com/open-feature/codegen/blob/main/.github/workflows/release.yaml#L50

Reply:
nice, done

Reply:
@beeme1mr note that I had to reverse the order of some steps for this, since we need to checkout before we install go, in order to read this file.